aws-athena/views/sale-vw_ias_salesval_upload.sql

```diff
@@ -0,0 +1,16 @@
+WITH ias_sales AS (
+    SELECT 
+        salekey,
+        NULLIF(REPLACE(instruno, 'D', ''), '') AS instruno_clean
+    FROM {{'iasworld', 'sales'}}
```

Comment:
**[Nitpick]** I think we'll need to wrap these strings in a [`source()` call](https://docs.getdbt.com/docs/build/sources) here to instruct dbt to properly reference the table! The [workflow](https://github.com/ccao-data/data-architecture/actions/runs/6384372564/job/17326897801?pr=147) isn't failing for this specific reason, but I expect if we resolved the error that the workflow run is reporting we would see another error get raised for this line.

```suggestion
    FROM {{ source('iasworld', 'sales') }}
```